### DISCUSSION TITLE
The efficient way to update?

### DISCUSSION THREAD

**Discussion by cutbypham** [chrismwilliams/astro-theme-cactus#46] [Q&A]
- I create a pull request to my source code and see the difference
- I don't know another way to update

**Comment by chrismwilliams** [comment]
Hi @thuanowa, yeah I've been asked this question before [here](https://github.com/chrismwilliams/astro-theme-cactus/issues/43#issuecomment-1412306012). Quick answer is it depends, but github does have some examples in its docs

**Reply by raj23689** [reply]
I got conflict errors while doing that. Look like some major changes have come.
Is there any way to create an efficient update, something like how Hugo Blowfish theme does [here](https://blowfish.page/docs/installation/#installing-updates)?
